book/src/how_to/building_drivers.md

```diff
@@ -11,10 +11,17 @@ value to `efi_boot_service_driver` or `efi_runtime_driver`.
 
 Example:
 
-```toml
-# In .cargo/config.toml:
-[build]
-rustflags = ["-C", "link-args=/subsystem:efi_runtime_driver"]
+```rust
+// In build.rs
+
+fn main() {
+    println!("cargo::rerun-if-changed=build.rs");
```

Comment:
I think this happens automatically

Reply:
Looks like it

> Cargo automatically handles whether or not the script itself needs to be recompiled, and of course the script will be re-run after it has been recompiled. Otherwise, specifying build.rs is redundant and unnecessary.